commerce/bookstore/python/todo.md

```diff
@@ -0,0 +1,8 @@
+# ToDo list
```

Comment:
Is this file meant to be checked in?

Reply:
Yeah, for the future me or future generations. 
I believe It's even mentioned in the readme.